werkzeug/serving.py

```diff
@@ -246,8 +258,16 @@ def write(data):
                     self.send_header('Date', self.date_time_string())
                 self.end_headers()
 
+                if 'transfer-encoding' in header_keys:
+                    for key, value in response_headers:
+                        if 'transfer-encoding' == key.lower():
```

Comment:
please don't do yoda style. it's awful.

Reply:
@ThiefMaster Do you mean to place the string on the right?

Reply:
Yes, `'constant' == variable` is yoda style